.appveyor.yml

```diff
@@ -37,4 +37,4 @@ build: false
 
 test_script:
 # Run tests. These include the equivalent of Travis R tests via test_r.py.
-- pytest --test-r --verbose
+- pytest --test-r --verbose --capture=tee-sys
```

Comment:
Is it preferable to leave this on permanently? Could you explain why?

Reply:
removed it